examples/platform_view/lib/main.dart

```diff
@@ -69,15 +67,15 @@ class _MyHomePageState extends State<MyHomePage> {
                   mainAxisAlignment: MainAxisAlignment.center,
                   children: <Widget>[
                     Text(
-                      'Button tapped $_counter time${ _counter == 1 ? '' : 's' }.',
+                      'Button tapped $_counter time${_counter == 1 ? '' : 's'}.',
                       style: const TextStyle(fontSize: 17.0),
                     ),
                     Padding(
                       padding: const EdgeInsets.all(18.0),
                       child: ElevatedButton(
                         child: Platform.isIOS
-                          ? const Text('Continue in iOS view')
-                          : const Text('Continue in Android view'),
+                            ? const Text('Continue in iOS view')
```

Comment:
Two space indents seem to have been good enough most everywhere else.

Reply:
Same reason as above.